packages/background/package.json

```diff
@@ -41,6 +43,12 @@
     "@ledgerhq/hw-transport": "^6.20.0",
     "@ledgerhq/hw-transport-webhid": "^6.20.0",
     "@ledgerhq/hw-transport-webusb": "^6.20.0",
+    "@keystonehq/bc-ur-registry-cosmos": "^0.0.2-alpha.1",
+    "@keystonehq/bc-ur-registry-eth": "^0.11.4",
+    "@keystonehq/base-eth-keyring": "^0.6.4",
+    "@keystonehq/cosmos-keyring": "^0.0.1-alpha.0",
+    "@keystonehq/bc-ur-registry": "^0.5.1",
+    "@cosmjs/crypto": "^0.29.3",
```

Comment:
In Keplr, we do not use @cosmjs in package.json.

Reply:
How can i use `@cosmjs/crypto`?